src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Functions/Processor.cs

```diff
@@ -127,6 +127,8 @@ ILogger<Processor> logger
                 _logger.LogError(ex, $"{GetType().Name} function FAILED for : Import: " +
                                      $"{message.Id} : {ex.Message}");
 
+                _logger.LogError(ex.StackTrace);
```

Comment:
Stacktrace should be available with previous log message I think, so is this needed?